Sources/PortalSwift/Components/PortalWebView.swift

```diff
@@ -126,11 +126,11 @@ public class PortalWebView: UIViewController, WKNavigationDelegate, WKScriptMess
       if let data = data as? [String: String] {
         print("Chain changed is parseable. \(data)")
 
-        let chainId = Int(UInt8(data["chainId"] ?? "0", radix: 16) ?? 0)
-        if chainId > 0 {
+        let chainIdString = data["chainId"] ?? "0" // Get the string value, defaulting to "0" if nil
```

Comment:
feels weird to default to chain Id 0. i know we did that before but shouldn't we just error?

Reply:
We should, when reviewing Blake's PRs let's make sure to include that (I mentioned the default chainId issue on his multi-wallet PRs as comments)